src/query_builder.erl

```diff
@@ -124,20 +122,21 @@ values_query(Collection, Metric, Namespace, Tag)
         "  WHERE collection = $1"
         "    AND metric = $2"
         "    AND dimensions ? $3",
-    Tag1 = encode_tag_key(Namespace, Tag),
+    Tag1 = dqe_idx_pg_utils:encode_tag_key(Namespace, Tag),
     Values = [Collection, Metric, Tag1],
     {ok, Query, Values}.
 
-lookup_query(Lookup, []) ->
+lookup_query(Lookup, _Start, _Finish, []) ->
     {Condition, CVals} = lookup_condition(Lookup),
-    Query = ["SELECT bucket, key FROM metrics WHERE " | Condition],
+    Query = ["SELECT bucket, key FROM metrics WHERE "
+             "time_range && " | Condition],
     {ok, Query, CVals};
-lookup_query(Lookup, KeysToRead) ->
+lookup_query(Lookup, _Start, _Finish, KeysToRead) ->
     {Condition, CVals} = lookup_condition(Lookup),
     I = length(CVals),
     Query = ["SELECT bucket, key, slice(dimensions, $", i2l(I + 1), ")"
              "  FROM metrics WHERE " | Condition],
-    Keys = [encode_tag_key(Ns, Name) || {Ns, Name} <- KeysToRead],
+    Keys = [dqe_idx_pg_utils:encode_tag_key(Ns, Name) || {Ns, Name} <- KeysToRead],
```

Comment:
Line 139 is too long:     Keys = [dqe_idx_pg_utils:encode_tag_key(Ns, Name) || {Ns, Name} <- KeysToRead],.